python/tests/test_files.py

```diff
@@ -0,0 +1,35 @@
+# Testing `feature_code_for`
+import pytest
+
+
+def test_feature_code_for_existing_example(examples, features_path, samples):
```

Comment:
If anything is tested here, it should be that there is at least 1 feature file and that for each feature file there is a matching ndjson file. Everything else seems spurious.

Reply:
Yep, the tests for compatibility should live in a compatibility folder in the python implementation.

See: https://github.com/cucumber/cucumber-ruby/blob/main/compatibility/support/shared_examples.rb

Reply:
From the looks of it he's copied the ruby structure which is similar to this. I've basically removed everything aside from the helper/tests for the helper